OKTA-528690 Add primaryFactor property for Signon Action object (#3632)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/reference/api/policy/index.md

| ---                     | ---                                                                                                                                                                       | ---                                             | ---                           | ---     |
| access                  | `ALLOW` or `DENY`                                                                                                                                                         | `ALLOW` or `DENY`                               | Yes                           | N/A     |
| requireFactor           | Indicates if multifactor authentication is required                                                                                                                      | Boolean                                         | No                            | false   |

> | primaryFactor <ApiLifecycle access="ie" /> | Indicates the primary factor used to establish a session for the org. Possible values: `PASSWORD_IDP_ANY_FACTOR` (any factor can be used to meet the authentication policy), `PASSWORD_IDP` (only a password can be used to meet the authentication policy)  | String | Yes, if `access` is set to `ALLOW`                        | N/A   |

Reviewer: Instead of `any factor can be used to meet the authentication policy` maybe should say `any factor used to meet the authentication policy of the app` - In OIE, you are always logging into some app (we refer to app sign on policies are as authentication policies). PASSWORD_IDP_ANY_FACTOR means anything the app's authentication policy requires can be used to establish a session.

I would then change `only a password can be used to meet the authentication policy` to `a password must always be used`.

> | primaryFactor <ApiLifecycle access="ie" /> | Indicates the primary factor used to establish a session for the org. Possible values: `PASSWORD_IDP_ANY_FACTOR` (any factor required by the app authentication policy can be used to establish a session), `PASSWORD_IDP` (a password must always be used to establish a session)  | String | Yes, if `access` is set to `ALLOW`                        | N/A   |

Reviewer 2: ```suggestion
| primaryFactor <ApiLifecycle access="ie" /> | Indicates the primary factor used to establish a session for the org. Supported values: `PASSWORD_IDP_ANY_FACTOR` (You can use any factor required by the app authentication policy to establish a session), `PASSWORD_IDP` (you must always use a password to establish a session)  | String | Yes, if `access` is set to `ALLOW`                        | N/A   |
```

Author: 🙇 will commit these suggestions with slight modification
